cpp/ql/src/Diagnostics/FailedExtractions.ql

```diff
@@ -0,0 +1,22 @@
+/**
+ * @kind diagnostic
```

Comment:
Per the spec (that was probably updated after you wrote the query), this is missing a name and description.

Reply:
Ta, added